app/src/validators/EmailValidator.py

```diff
@@ -0,0 +1,12 @@
+from email_validator import validate_email as check_email, EmailNotValidError
+from wtforms import ValidationError
+
+
+class EmailValidator():
+
+    def __init__(self, message="Email is not valid"):
+        self.message = message
+
+    def __call__(self, form, field):
+
+        check_email(field.data)
```

Comment:
maybe it is worth to paste  check_email contents here and then remove check_email? or are you reusing check_email method somewhere?

Reply:
that method is from email_validation module, I use somebody's module to validate email.